server/utils/cookieless_utils.ts

```diff
@@ -94,10 +94,14 @@ const acquireEmbedSessionInternal = async (
   session_reference_token?: string
 ) => {
   try {
-    const request = {
+    // TODO type when 23.8 Looker SDK is published
+    const request: any = {
       ...user,
       session_reference_token: session_reference_token,
     }
+    if (config.use_embed_domain) {
+      request.embed_domain = `http://${config.demo_host}:${config.demo_port}`
```

Comment:
easier to use http for the demo?

Reply:
Demo server uses and currently does not have a mechanism to override it. This is demo code to show how use it. I've added comments to call the simplicity of the implementation out.